webserver/templates/datasets/evaluate.html

```diff
@@ -21,6 +21,10 @@ <h2 class="page-title">Evaluate dataset "{{ dataset['name'] }}"</h2>
         <div class="col-sm-4">{{ form.filter_type(class="form-control", required="required") }}</div>
       </div>
       <div class="form-group">
+        <label class="col-sm-2 control-label">{{ form.evaluation_location.label.text }}</label>
+        <div class="col-sm-4">{{ form.evaluation_location(class="form-control", required="required") }}</div>
+      </div>
+      <div class="form-group">
```

Comment:
It would be nice to show some documentation here for users who don't know what this is. This should be another component of the project that we add. I'm not sure what the best way to do this is. One option is to make our own help pages and link to them from here, or to add our documentation to the github wiki for acousticbrainz-server. What do you think?